### PR TITLE
Takeoff documentation

### PR DESCRIPTION
**Pull request by christophe-david**
This PR updates Sphinx doc regarding recent takeoff additions.
Preview available here : https://fast-oad.readthedocs.io/en/takeoff-documentation/documentation/mission_module/mission_file/segments.html

Also, some fixes in documentation have been added.

### CODE REVIEW

File: docs/documentation/mission_module/mission_file/segments.rst [hunk 1]
```diff
@@ -389,12 +408,141 @@ Typically, it will be used as last segment to compute a reserve based on the Zer
...
+.. code-block:: yaml
+
+    segment: takeoff
+    wheels_friction: 0.03
+    rotation_equivalent_airspeed:
+        value: data:mission:operational:takeoff:Vr
+    rotation_rate:
+      value: 0.0523
+      units: rad
+    rotation_alpha_limit_truc:
```
Comment (esnguyenvan) [inline docs/documentation/mission_module/mission_file/segments.rst:533]: The name of this parameter sounds wrong, or still in a 'temporary' state.

**Review by esnguyenvan — CHANGES_REQUESTED** [review]
Thanks for updating the documentation. I just noticed the name of one takeoff parameter sounds wrong. The rest seems ok to me.

File: docs/documentation/mission_module/mission_file/segments.rst (hunk 1 above)
Comment (christophe-david) [reply docs/documentation/mission_module/mission_file/segments.rst]: Oops. Fixed in 0939b08c667df62a2f01f51e563f86cab95f131c with some other minor problems.

**Review by esnguyenvan — APPROVED** [review]
Great

**Pull request merged**